js/apps/admin-ui/src/util.ts

```diff
@@ -173,7 +173,7 @@ export const generateId = () => Math.floor(Math.random() * 1000);
 export const localeToDisplayName = (locale: string, displayLocale: string) => {
   try {
     return new Intl.DisplayNames([displayLocale], { type: "language" }).of(
-      locale,
+      locale === "zh-CN" ? "zh-HANS" : locale === "zh-TW" ? "zh-HANT" : locale,
```

Comment:
Perhaps we could add a comment here with some explainer why this is needed for future reference.

Reply:
will be part of next commit